packages/manager/modules/iam/src/routes/policyAdd.route.js

```diff
@@ -0,0 +1,24 @@
+import { policyAdd } from '@iam/components';
+import { asParams, asResolve } from '@iam/resolves';
+
+export const name = 'policyAdd';
```

Comment:
I know this is a pain but
Can we rename this to
```suggestion
export const name = 'createPolicy';
```

Reply:
It's fixed.